server/routes/authorRoute.js

```diff
@@ -10,5 +10,12 @@ authorRoute.get('/', Authenticate.isLoggedIn, AuthorController.listAuthor);
 authorRoute.get('/:id', AuthorController.getAuthor);
```

Comment:
This route was supposed to be protected.

Reply:
For the Admin alone?

Reply:
For logged in users